### PR TITLE
Gs 22/gs 114/read me service account

### PR DESCRIPTION
**Pull request by Mehul-Gandhi**
### Jira Ticket

[Jira Ticket](https://dashbd.atlassian.net/browse/GS-114?focusedCommentId=10116&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-10116)

### Description

Add README steps for configuring .env variables. Add instructions on creating a service account. Clarify instructions on gradescope email and password `.env` variables.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Changes

1. Update README
2. Include instructions on getting a gradescope password
3. Include instructions on creating a GCP service account
### Testing

No testing necessary for a README. 
### Checklist

- [x] My branch name matches the format: `<ticket-id>/<brief-description-of-change>`
- [x] My PR name matches the format: `[<ticket-id>] <brief-description-of-change>`
- [ ] I have added doc-comments to all new functions ([JSDoc](https://jsdoc.app/) for JS and [Docstrings](https://peps.python.org/pep-0257/) for Python)
- [x] I have reviewed all of my code
- [x] My code only contains major changes related to my ticket

### CODE REVIEW

**Review by naveen-nathan — APPROVED** [review]
Good job making this README, Mehul! It is very easy to understand.

File: gradescopeCronJob/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# README
+
+- To use this script, your account credentials must be a teaching assistant or instructor on a CS10 Gradescope course instance.
+
+### 1. Environment Setup
+- Store GradeScope credentials in environment variables `EMAIL` and `PASSWORD`. Define these in a `.env` file.
```
Comment (Mehul-Gandhi) [inline gradescopeCronJob/README.md]:
```suggestion
- Store Gradescope credentials in environment variables `EMAIL` and `PASSWORD`. Define these in a `.env` file.
```

File: gradescopeCronJob/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- If you do not have a password setup on GradeScope, you will need to create a password.
```
Comment (Mehul-Gandhi) [inline gradescopeCronJob/README.md]:
```suggestion
- If you do not have a password setup on Gradescope, you will need to create a password.
```

File: gradescopeCronJob/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+     SERVICE_ACCOUNT_CREDENTIALS=<contents-of-downloaded-keyfile>
+     ```
+   - Example:
+    ```plaintext
+    SERVICE_ACCOUNT_CREDENTIALS={"type":"service_account","project_id":"your-project-id","private_key_id":"your-private-key-id","private_key":"-----BEGIN PRIVATE KEY-----\\nMIIEvAIBADANBgkq...\\n-----END PRIVATE KEY-----\\n","client_email":"your-service-account-email@your-project-id.iam.gserviceaccount.com","client_id":"your-client-id","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url":"https://www.googleapis.com/robot/v1/metadata/x509/your-service-account-email%40your-project-id.iam.gserviceaccount.com"}
+    ```
+
+### 3. Set Constants as Necessary
+
+- **COURSE_ID**: The course ID is the final component of the URL on the GradeScope course homepage: `https://www.gradescope.com/courses/[COURSE_ID]`
```
Comment (Mehul-Gandhi) [inline gradescopeCronJob/README.md]:
```suggestion
- **COURSE_ID**: The course ID is the final component of the URL on the Gradescope course homepage: `https://www.gradescope.com/courses/[COURSE_ID]`
```

**Pull request merged**